ui.py

```diff
@@ -25,13 +25,13 @@ def show_books(books):
      :param books: the book list """
 
     # TODO print a blank line before and after the book list or no books message
```

Comment:
This will no longer be a TODO item. The comment on line 27 can be removed.

Reply:
@yp0975eu  I have removed the todo comment

Reply:
I still see it.